### PR TITLE
Add filter, sort, limit and offset to rest api guide

### PR DESCRIPTION
**Pull request by janbuecker**

### CODE REVIEW

File: source/developers-guide/rest-api/index.md [hunk 1]
```diff
@@ -242,7 +242,141 @@ var string = "2012-06-13T07:32:25.706Z"
...
+**Example: Active articles with at least 1 pseudo sale**
+
+```
+$params = [
+    'filter' => [
+        [
+	        'property' => 'pseudoSales',
+	        'expression' => '>=',
+	        'value' => 1
+	    ],
```
Comment (mitelg) [inline source/developers-guide/rest-api/index.md]: indenting is wrong

File: source/developers-guide/rest-api/index.md [hunk 2]
```diff
@@ -242,7 +242,141 @@ var string = "2012-06-13T07:32:25.706Z"
...
+**Example: Active articles or articles containing "beach" in their name**
+
+```
+$params = [
+    'filter' => [
+        [
+	        'property' => 'name',
+	        'value' => '%beach%',
+	        'operator' => 1
+	    ],
```
Comment (mitelg) [inline source/developers-guide/rest-api/index.md]: indenting is wrong

File: source/developers-guide/rest-api/index.md [hunk 3]
```diff
@@ -242,7 +242,141 @@ var string = "2012-06-13T07:32:25.706Z"
...
+	        'property' => 'active',
+	        'value' => 1
+	    ]
+    ]
+];
+
+$client->get('articles', $params);
+```
+
+**Example: Supplier is "shopware"**
```
Comment (mitelg) [inline source/developers-guide/rest-api/index.md]: does not match with the example below

**Comment by mitelg** [comment]
I think this PR #328 is obsolete if this PR is merged

**Comment by janbuecker** [comment]
Yes, PR #328 can be closed afterwards.

**Pull request merged**